src/hint_processor/hint_utils.zig

```diff
@@ -341,3 +367,94 @@ test "getPtrFromVarName: invalid" {
         getPtrFromVarName("value", &vm, ids_data, .{}),
     );
 }
+
+test "getRelocatableFromVarName: valid" {
+    // Initializes the Cairo virtual machine.
+    var vm = try CairoVM.init(std.testing.allocator, .{});
+    defer vm.deinit(); // Ensure cleanup.
+
+    // Sets up memory segments in the virtual machine with an invalid configuration.
+    try vm.segments.memory.setUpMemory(
+        std.testing.allocator,
+        .{.{ .{ 1, 0 }, .{0} }},
+    );
+    defer vm.segments.memory.deinitData(std.testing.allocator); // Clean up memory data.
+
+    // Creates a hashmap containing variable references.
+    var ids_data = std.StringHashMap(HintReference).init(std.testing.allocator);
+    defer ids_data.deinit();
+
+    // Inserts a valid variable reference into the hashmap.
+    try ids_data.put("value", HintReference.initSimple(0));
+
+    // Invokes `getPtrFromVarName` to retrieve the pointer from the "value" variable, expecting an error.
+    try expectEqual(
+        Relocatable.init(1, 0),
+        try getRelocatableFromVarName("value", &vm, ids_data, .{}),
+    );
+}
+
+test "getRelocatableFromVarName: invalid" {
+    // Initializes the Cairo virtual machine.
+    var vm = try CairoVM.init(std.testing.allocator, .{});
+    defer vm.deinit(); // Ensure cleanup.
+
+    // Sets up memory segments in the virtual machine with an invalid configuration.
+    try vm.segments.memory.setUpMemory(
+        std.testing.allocator,
+        .{.{ .{ 1, 0 }, .{0} }},
+    );
+    defer vm.segments.memory.deinitData(std.testing.allocator); // Clean up memory data.
+
+    // Creates a hashmap containing variable references.
+    var ids_data = std.StringHashMap(HintReference).init(std.testing.allocator);
+    defer ids_data.deinit();
+
+    // Inserts a valid variable reference into the hashmap.
+    try ids_data.put("value", HintReference.initSimple(-8));
+
+    // Invokes `getPtrFromVarName` to retrieve the pointer from the "value" variable, expecting an error.
+    try expectEqual(
+        HintError.UnknownIdentifier,
+        getRelocatableFromVarName("value", &vm, ids_data, .{}),
+    );
+}
+
+test "getConstantFromVarName: valid without point in the middle of var name" {
```

Comment:
isn't it the other way around? with / without the period?

Reply:
Don't think so because we have to handle the point case which means that for `ids.MAX_LOW` we must capture `MAX_LOW` (which is tested in the following unit test). In this specific test, I am testing the simplest case (without any point in the middle) with only `value` which must be captured correctly. I also added `ids.MAX_LOW` in the hashmap but it is of no use, just to check that it does not pollute the test.